### PR TITLE
Add weekly list

### PR DESCRIPTION
**Pull request by jaredoong**
User unable to keep track of recipes that they plan to use for the week.

Let's:
* Add functionality to create a weekly recipes planner
* Allow viewing of weekly plan
* Allow adding of recipes into weekly plan (name + quantity)
* Allow deleting of recipes from weekly plan

### CODE REVIEW

File: src/main/java/seedu/meal360/Parser.java [hunk 1]
```diff
@@ -116,10 +87,41 @@ public RecipeList parseListRecipe(String[] inputs, RecipeList recipeList) {
...
+        int numDays = 0;
+        if (command[1].equals("/add")) {
+            numDays = Integer.parseInt(command[command.length - 1]);
+            if (numDays < 1) {
+                throw new NumberFormatException();
+            }
+        }
+
+        int nameLastIndex = (command[1].equals("/add")) ? command.length - 1 : command.length;
+        WeeklyPlan addToPlan = new WeeklyPlan();
```
Comment (JuneNita) [inline src/main/java/seedu/meal360/Parser.java:111]: i think it's better to change the variable name eg. 
 `thisWeekRecipes`, `plannedRecipes`,  `recipesPlan`

Comment (JuneNita) [inline src/main/java/seedu/meal360/Parser.java:111]: maybe can change the variable name eg, `thisWeekPlan`, `recipesPlan`, `newRecipePlan`

File: src/main/java/seedu/meal360/Parser.java [hunk 2]
```diff
@@ -5,16 +5,9 @@
...
 
-    private static final Ui ui = new Ui();
-
-    public String[] parseCommand(String command) {
-        return command.split(" ");
-    }
-
     public Recipe parseAddRecipe(String[] input, RecipeList recipeList) {
-
         StringBuilder recipeName = new StringBuilder(input[2]);
```
Comment (TopGun2001) [inline src/main/java/seedu/meal360/Parser.java:9]: I also used StringBuilder to combine words. This is very helpful.

File: src/main/java/seedu/meal360/Parser.java [hunk 3]
```diff
@@ -87,24 +76,6 @@ public Recipe parseDeleteRecipe(String[] input, RecipeList recipeList) {
         }
     }
 
-    public void parseHelpUser(){
```
Comment (TopGun2001) [inline src/main/java/seedu/meal360/Parser.java:90]: It's good to add a help command as input to assist the user in commands. We also need to add this in user guide.

**Pull request merged**